README.md

```diff
@@ -5,4 +5,76 @@
 [![Go Report Card](https://goreportcard.com/badge/stackanetes/kubernetes-entrypoint "Go Report Card")](https://goreportcard.com/report/stackanetes/kubernetes-entrypoint)
 
 
+============
+
+Kubernetes-entrypoint enables complex deployments on top of Kubernetes.
+
+## Overview
+
+Kubernetes-entrypoint needs to be deployed as a container entrypoint which means it has to bundled in container. Before launching desired application
```

Comment:
`Kubernetes-entrypoint is meant to be used as a container entrypoint, which means it has to bundled in the container. Before launching the desired application, the entrypoint verifies and waits for all specified dependencies to be met.`